pkg/controller/services.go

```diff
@@ -1321,6 +1327,23 @@ func (cont *AciController) opflexDeviceChanged(obj apicapi.ApicObject) {
 	domName := obj.GetAttrStr("domName")
 	ctrlrName := obj.GetAttrStr("ctrlrName")
 
+	if strings.Contains(cont.config.Flavor, "openstack") {
+		compHvDn := obj.GetAttrStr("compHvDn")
+		if strings.Contains(compHvDn, "prov-OpenStack") {
+			opflexOdevInfo, ok := cont.openStackFabricPathDnMap[obj.GetAttrStr("hostName")]
+			if ok {
+				opflexOdevInfo.count++
+				cont.openStackFabricPathDnMap[obj.GetAttrStr("hostName")] = opflexOdevInfo
+			} else {
+				var openstackopflexodevinfo openstackOpflexOdevInfo
+				openstackopflexodevinfo.count++
+				openstackopflexodevinfo.fabricPathDn = obj.GetAttrStr("fabricPathDn")
+				openstackopflexodevinfo.dn = obj.GetAttrStr("dn")
+				cont.openStackFabricPathDnMap[obj.GetAttrStr("hostName")] = openstackopflexodevinfo
+			}
+			cont.updateDeviceCluster()
```

Comment:
given that we dont anticipate too many opflex add events for the openstack nodes, should we log here at info level to indicate that we are processing at this point?

Reply:
Added log